nilearn/datasets/struct.py

```diff
@@ -164,17 +198,69 @@ def load_mni152_brain_mask():
     """
     # Load MNI template
     target_img = load_mni152_template()
-    mask_voxels = (get_data(target_img) > 0).astype(int)
+    mask_voxels = (get_data(target_img) > threshold).astype(int)
     mask_img = new_img_like(target_img, mask_voxels)
     return mask_img
 
 
+def load_mni152_gm_mask(threshold=0.):
+    """Load brain mask from gray-matter MNI152 template
+
+    Parameters
+    ----------
+    threshold : float, optional
+        The parameter which amounts to include the values in the mask image.
+        Values lying above this threshold will be included. Default=0.2
+
+    Returns
+    -------
+    mask_img : Nifti-like mask image corresponding to gray matter.
+
+    """
+    # Load MNI template
+    gm_target = load_mni152_gm_template()
+    gm_target_img = check_niimg(gm_target)
+    gm_target_data = niimg._safe_get_data(gm_target_img)
+
+    gm_target_mask = (gm_target_data > threshold)
+
+    gm_target_mask = ndimage.binary_closing(gm_target_mask, iterations=2)
+    gm_target_mask_img = new_img_like(gm_target_img, gm_target_mask)
+
+    return gm_target_mask_img
+
+
+def load_mni152_wm_mask(threshold=0.):
+    """Load brain mask from white-matter MNI152 template
+
+    Parameters
+    ----------
+    threshold : float, optional
+        The parameter which amounts to include the values in the mask image.
+        Values lying above this threshold will be included. Default=0.2
```

Comment:
.2  or  0?
I'm wondering: Is this function meant to be used directly (public API) or an internal function ?

Reply:
I changed the docstring to 0. With a quick `git grep load_mni152_wm_mask`, it seems that this function is only used internally.